USB/mcc-libusb/usb-2416.c

```diff
@@ -251,7 +258,8 @@ int usbAIn_USB2416(libusb_device_handle *udev, uint8_t channel, uint8_t mode, ui
     return -1;
   }
 
-  libusb_control_transfer(udev, requesttype, AIN, wValue, wIndex, (unsigned char *) &data, sizeof(data), HS_DELAY);
+  if (libusb_control_transfer(udev, requesttype, AIN, wValue, wIndex, (unsigned char *) &data, sizeof(data), HS_DELAY) < 0)
```

Comment:
Above you returned whatever the return value was. Here, you just return -1. Is there a reason for this?

Reply:
Also below in `usbReadMemory_USB2416_Error_Return`, you just return the return value as well.

Reply:
Yes. This is the function you and I were talking about that returns an analog reading, where -1 can correspond, in theory, to a valid return value. I want to return as few values as possible that can correspond to an error code, to minimize the potential amount of false negative readings.